data/strings/types_strings.txt

```diff
@@ -24478,6 +24479,7 @@
     et = Haagiselamute müük
     fi = Matkailu- ja retkeilyautokauppias
     fr = Concessionnaire de caravanes et camping-cars
+    he = סוחר בקראוונים
```

Comment:
Sounds like RV *dealer* instead of *dealership*, but can't think of a better term and google translate can't either